Marlin/src/module/planner.cpp

```diff
@@ -2063,21 +2104,31 @@ bool Planner::_populate_block(block_t * const block, bool split_move,
       steps_dist_mm.c = dc * mm_per_step[C_AXIS],
       steps_dist_mm.i = di * mm_per_step[I_AXIS],
       steps_dist_mm.j = dj * mm_per_step[J_AXIS],
-      steps_dist_mm.k = dk * mm_per_step[K_AXIS]
+      steps_dist_mm.k = dk * mm_per_step[K_AXIS],
+      steps_dist_mm.u = du * mm_per_step[U_AXIS],
+      steps_dist_mm.v = dv * mm_per_step[V_AXIS],
+      steps_dist_mm.w = dw * mm_per_step[W_AXIS]
     );
   #endif
 
   TERN_(HAS_EXTRUDERS, steps_dist_mm.e = esteps_float * mm_per_step[E_AXIS_N(extruder)]);
 
   TERN_(LCD_SHOW_E_TOTAL, e_move_accumulator += steps_dist_mm.e);
 
+  #if HAS_ROTATIONAL_AXES
+    bool cartesian_move = true;
+  #endif
```

Comment:
This doesn't seem to ever be consumed. Was this only here for debugging?

Reply:
I thought I would need it later, but now I removed this variable